sycl/include/CL/sycl/feature_test.hpp

```diff
@@ -15,6 +15,8 @@ namespace sycl {
 // TODO: Move these feature-test macros to compiler driver.
 #define SYCL_EXT_INTEL_DEVICE_INFO 2
 #define SYCL_EXT_ONEAPI_MATRIX 1
+#define SYCL_EXT_ONEAPI_MAX_GLOBAL_NUMBER_WORK_GROUPS 1
+#define SYCL_EXT_ONEAPI_MAX_NUMBER_WORK_GROUPS 1
```

Comment:
We should not have two feature-test macros for this feature.  Each feature has just one feature-test macro.  Choose a single feature-test macro name that makes sense for both of the "info" descriptors you are adding.  For example, maybe `SYCL_EXT_ONEAPI_WORK_GROUP_QUERY` or `SYCL_EXT_ONEAPI_WORK_GROUP_SIZE_QUERY`.